data/adoptions.yaml

```diff
@@ -1,20 +1,18 @@
 main:
-- url: https://github.com/awslabs/tough
+- url: https://github.com/cnabio/cnab-spec/blob/cnab-security-1.0.0-ga/300-CNAB-security.md
+  tag: CNAB
+- url: https://github.com/bottlerocket-os/bottlerocket/tree/develop/sources/updater
   tag: Amazon
-- url: https://fuchsia.dev/
+- url: https://fuchsia.dev/fuchsia-src/concepts/system/software_update_system
```

Comment:
Should we change the logo to show the google fuchsia logo instead of just the google logo?